js/forum/src/components/EditUserModal.js

```diff
@@ -37,22 +38,32 @@ export default class EditUserModal extends Modal {
       <div className="Modal-body">
         <div className="Form">
           <div className="Form-group">
-            <label>Username</label>
+            <label>{app.translator.trans('core.forum.edit_user.username_heading')}</label>
             <input className="FormControl" placeholder={extractText(app.translator.trans('core.forum.edit_user.username_label'))}
               bidi={this.username} />
           </div>
 
           {app.session.user !== this.props.user ? [
             <div className="Form-group">
-              <label>Email</label>
+              <label>{app.translator.trans('core.forum.edit_user.email_heading')}</label>
               <div>
                 <input className="FormControl" placeholder={extractText(app.translator.trans('core.forum.edit_user.email_label'))}
                   bidi={this.email} />
               </div>
+              {!this.isActivated() ? (
+                <div>
+                  {Button.component({
+                    className: 'Button Button--block',
+                    children: app.translator.trans('core.forum.edit_user.activate_button'),
+                    loading: this.loading,
+                    onclick: this.activate.bind(this)
+                  })}
+                </div>
+              ) : ''}
```

Comment:
Interesting... is there reasoning behind this? It seems a bit confusing for "if is actived show nothing else show this button" instead of "if is not activated show button"